### PR TITLE
add String blitting benchmarks

### PR DESCRIPTION
**Pull request by weissi**
Motivation:

Writing Strings to ByteBuffers is important, let's have some more
benchmarks.

Modifications:

add String to ByteBuffer benchmarks

Result:

more benchmarks

### CODE REVIEW

**Comment by weissi** [comment]
spurious fail

```
Posix wrapper adds more than 20% overhead (with wrapper: 0.804463982582092, without: 0.661388993263245
```

**Comment by weissi** [comment]
@swift-nio-bot test this please

File: Sources/NIOPerformanceTester/main.swift [hunk 1]
```diff
@@ -174,8 +180,114 @@ final class RepeatedRequests: ChannelInboundHandler {
...
+    for i in 1..<10 {
+        headers.append(("\(i)", "\(i)"))
+    }
+
+    var val = 0
+    for _ in 0..<100_000 {
+        let headers = HTTPHeaders(headers)
+        val += headers.underestimatedCount
+    }
+    //precondition(val == 1_000_000)
```
Comment (Lukasa) [inline Sources/NIOPerformanceTester/main.swift]: Do we need this?

Comment (weissi) [reply Sources/NIOPerformanceTester/main.swift]: nope, thanks! fixed

**Pull request merged**